### DISCUSSION TITLE
Error related to json request parsing

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4382] [Q&A] [answered]
Hello everyone, I have a question, I'm using json::parsing to download information and get the following error
![msg5513425873-1091 000006](https://github.com/nlohmann/json/assets/36815965/3d4f93cc-6f13-4673-8ef5-69715f5c41bb)

![image](https://github.com/nlohmann/json/assets/36815965/a3078b13-f724-4792-9b7a-d2869469e8d6)

![image](https://github.com/nlohmann/json/assets/36815965/35997866-1f53-4b59-bfdf-5796330faa7d)

The JSON itself for parsing looks like this
```json
{
    "from": "products",
    "limit": "",
    "method": "update",
    "order": "",
    "vars": {
        "price": "80",
        "items": "3"
    },
    "where": "id IN('44852')"
}
```

At the same time, JSON is correct, and the response JSON (the whole thing is involved in sending the request body to the server) and always looks like this

```json
{
    "status": "succes"
}
```

What could be the reason for such an error?

**Comment by nlohmann** [comment, marked as answer]
The `what()` message of the parse error exception should give you an idication of the byte where parsing fails.
